### DISCUSSION TITLE
Comments coloring

### DISCUSSION THREAD

**Discussion by genebean** [catppuccin/nvim#631] [Q&A] [answered]
I really like how this looks, with the exception of the comments... I find them hard to read. Can anyone enlighten me as to how to go about adjusting their color and/or contrast with the background?

![Screenshot 2023-12-20 at 4 23 40 PM](https://github.com/catppuccin/nvim/assets/966789/5f7edd38-e200-41f9-8d44-33427c702a2e)

**Comment by mrtnvgr** [comment, marked as answer]
```lua
require("catppuccin").setup {
    custom_highlights = function(colors)
        return {
            Comment = { fg = colors.flamingo },
        }
    end
}
```
